account-integrations/sdk/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@getwax/sdk",
```

Comment:
Thought this could be a good name to deploy it to

Reply:
Like the name 👍